samples/unity_v250_k8s_122.yaml

```diff
@@ -5,12 +5,21 @@ metadata:
   namespace: test-unity
```

Comment:
@panigs7 I think this file is not needed

Reply:
Removed the file